pkg/utils/utils.go

```diff
@@ -39,6 +39,10 @@ import (
 	"github.com/fission/fission/pkg/utils/uuid"
 )
 
+const (
+	ENV_DISABLE_OWNER_REFERENCES string = "DISABLE_OWNER_REFERENCES"
```

Comment:
Please add a method in utils

func IsOwnerReferencesEnabled() bool {
       return os.Getenv(utils.ENV_DISABLE_OWNER_REFERENCES) != "false" 
      // use parse bool or something more sophisticated here so that we can identify 
      // False, FASLE, false etc as well. As this directly gets input from chart.
}```

Once above done. 

- We should add enable_owner_references flags at struct level whereever it used. Just call util.IsOwnerReferencesEnabled function to get global value.
- Pass flag for functions which are not methods.
- Idea is to not read env variable every time only at struct creation time and every component has its own control.